pwndbg/commands/pie.py

```diff
@@ -98,30 +99,30 @@ def piebase(offset=None, module=None) -> None:
         print(message.error("Could not calculate VA on current target."))
 
 
-parser = argparse.ArgumentParser()
-parser.description = "Break at RVA from PIE base."
-parser.add_argument("offset", nargs="?", default=0, help="Offset to add.")
-parser.add_argument(
-    "module",
-    type=str,
-    nargs="?",
-    default="",
-    help="Module to choose as base. Defaults to the target executable.",
-)
-
-
-@pwndbg.commands.ArgparsedCommand(parser, aliases=["brva"], category=CommandCategory.BREAKPOINT)
-@pwndbg.commands.OnlyWhenRunning
-def breakrva(offset=0, module=None) -> None:
-    offset = int(offset)
-    if not module:
-        # Note: we do not use `pwndbg.gdblib.file.get_file(module)` here as it is not needed.
-        # (as we do need the actual path that is in vmmap, not the file itself)
-        module = get_exe_name()
-    addr = translate_addr(offset, module)
-
-    if addr is not None:
-        spec = "*%#x" % (addr)
-        gdb.Breakpoint(spec)
-    else:
-        print(message.error("Could not determine rebased breakpoint address on current target"))
+if pwndbg.dbg.is_gdblib_available():
```

Comment:
Random thought: Wouldnt it be better/easier to just not import the command file if its not available at all?

Or is it that there could be other deps to it? I think there should never be such deps :X

Reply:
Well, the problem is that `breakrva` is the only command in that module that can't run without GDB. `piebase` can. If we want to avoid importing the command modules entirely, we would probably have to split the commands that can run and those that can't into separate modules, whenever a module has both.